src/routes/manga/+layout.svelte

```diff
@@ -0,0 +1,133 @@
+                                            class="h-full w-full rounded-[0.4rem] border-none bg-surface-400 pl-12 text-sm font-semibold text-white shadow-lg !ring-0 placeholder:font-medium placeholder:text-surface-200"
+                                        />
+                                    </form>
+                                </search-form>
+                            </div>
+                        {/if}
+                    </div>
+
+                    {#if $is_authenticated}
+                        <button
+                            class="avatar"
+                            aria-label="Avatar"
+                            use:tippy={{
+                                trigger: "click",
+                                arrow: false,
+                                allowHTML: true,
+                                placement: "bottom-end",
+                                animation: "shift-away",
+                                appendTo: document.body,
+                                interactive: true,
+                                theme: "elaine",
+                                onTrigger: async (instance) => {
+                                    // set "1 vw"
+                                    instance.props.offset = [0, vw(1)];
+                                    const node = document.createElement("avatar-tippy");
+                                    new ProfileDropdown({
+                                        target: node
+                                    });
+
+                                    instance.setContent(node);
+                                }
+                            }}
+                        >
+                            <Avatar
+                                rounded="rounded-[0.4rem] md:rounded-[0.375vw]"
+                                width="w-12 md:w-[2.5vw]"
+                                src="/images/Avatar.avif"
+                                initials="JD"
+                            />
+                        </button>
+                    {:else}
+                        <auth-buttons class="flex items-center md:gap-[0.75vw]">
+                            <a
+                                href="/user/register"
+                                class="btn hidden bg-surface-400 font-semibold leading-none md:flex md:rounded-[0.5vw] md:px-[1.25vw] md:py-[0.85vw] md:text-[1vw]"
+                            >
+                                Register
+                            </a>
+                            <a
+                                href="/user/login"
+                                class="btn rounded-[0.4rem] bg-primary-500 px-5 py-4 text-base font-semibold leading-none md:rounded-[0.5vw] md:px-[1.25vw] md:py-[0.85vw] md:text-[1vw]"
+                            >
+                                Login
+                            </a>
+                        </auth-buttons>
+                    {/if}
+                {/if}
+            </navbar>
```

Comment:
This is repeated logic no? Use a named layout

Reply:
yea I think so

Reply:
Lttm

Reply:
> but this structure is bit different so what do you think?

what do you mean by this actually?